push_data.yml

```diff
@@ -10,7 +10,10 @@
     - name: "db_user"
       prompt: "What database user should we use?"
       default: postgres
+    - name: "db_password"
+      prompt: "What database user should we use?"
```

Comment:
This should probably be `What password should we use for the database user?` or something, right?